internal/pkg/daemon/bpfrecorder/bpfrecorder_linux_ppc64le.go

```diff
@@ -0,0 +1,44 @@
+//go:build ppc64le && linux
```

Comment:
```suggestion
//go:build (ppc64le || s390x) && linux
```
What about something like the above?